src/pages/partners/_premium_agent.module.scss

```diff
@@ -0,0 +1,77 @@
+@use 'features/styles/theme/theme-mixins' as *;
+@import 'features/styles/theme/variables';
+
+
+.wrap_container {
```

Comment:
@konstantinosG-deriv I believe we still can use our our Flex component for this part we should avoid having css changes like this that we already have , we wanna do minimum css as we have our flexible components thanks 🙏 

for the media query you are using also we can have `md` , `lg` , please check it in the code base where we are using , if any questions or doubt lets do it together . Thanks 🙏

Reply:
Thanks @mitra-deriv, I used Flexbox.Box with **md** and removed the unnecessary scss